### PR TITLE
docs: Rewrite issue templates for false positives and false negatives

### PR DESCRIPTION
**Pull request by dune73**
I have rewritten the two issue templates from scratch. I thought it was necessary to make it clearer what we really need.

### CODE REVIEW

**Review by fzipi — DISMISSED** [review]
Way better, thanks.

File: .github/ISSUE_TEMPLATE/01_false-positive.md [hunk 1]
```diff
@@ -1,38 +1,68 @@
...
+<!--
+Feel free to skip this section if you provided a curl call above.
+
+Ideally, you provide a full audit log of the request, relevant infos out of
+the error log or at least a screenshot where we can see the payload so we
+can reproduce the behavior.
+
+Usually, you find the logs at a location like /var/log/modsec_audit.log.
+When using a CDN or cloud server, the naming of the logs and their location
+depends on the provider. Please refer to the documentation.
```
Comment (theseion) [inline .github/ISSUE_TEMPLATE/01_false-positive.md:51]:
```suggestion
depends on the provider. Please refer to their documentation.
```

File: .github/ISSUE_TEMPLATE/01_false-positive.md [hunk 2]
```diff
@@ -1,38 +1,68 @@
...
+
+If you can not submit a curl call nor log files or a payload to reproduce
```
Comment (theseion) [inline .github/ISSUE_TEMPLATE/01_false-positive.md:53]:
```suggestion
If you cannot submit neither curl call nor log files nor a payload to reproduce
```

File: .github/ISSUE_TEMPLATE/01_false-positive.md [hunk 3]
```diff
@@ -1,38 +1,68 @@
...
+the behavior, there is litterally nothing we can do for you. Please help us to
+the information we need to help you.
```
Comment (theseion) [inline .github/ISSUE_TEMPLATE/01_false-positive.md:55]:
```suggestion
get access to the information we need to help you.
```

File: .github/ISSUE_TEMPLATE/01_false-positive.md [hunk 4]
```diff
@@ -1,38 +1,68 @@
...
+<!-- Please provide all relevant information about your environment. -->
 
-* CRS version (e.g., v3.2.0):
-* Paranoia level setting:
-* ModSecurity version (e.g., 2.9.3):
-* Web Server and version (e.g., apache 2.4.41):
+* CRS version (e.g., v3.3.4):
+* Paranoia level setting (e.g. PL1) :
+* ModSecurity version (e.g., 2.9.6):
+* Web Server and version or cloud provider / CDN (e.g., apache 2.4.54):
```
Comment (theseion) [inline .github/ISSUE_TEMPLATE/01_false-positive.md:65]:
```suggestion
* Web Server and version or cloud provider / CDN (e.g., Apache httpd 2.4.54):
```

File: .github/ISSUE_TEMPLATE/02_false-negative.md [hunk 5]
```diff
@@ -1,30 +1,68 @@
 ---
 name: 'False negative'
```
Comment (theseion) [inline .github/ISSUE_TEMPLATE/02_false-negative.md:2]: Same changes as in the other template.

**Review by theseion — CHANGES_REQUESTED** [review]
Very good, thanks!

**Review by theseion — APPROVED** [review]

**Comment by dune73** [comment]
@RedXanadu Are you OK with this wording? If yes, please merge.

**Comment by RedXanadu** [comment]
Sorry for the delay in reviewing this…

@dune73 This looks good! The expanded guidance is clearer.

One comment I would make, from a _Dev on Duty_ perspective: if someone is reporting an FP then logs are a "must have", while a curl call is only a "nice to have". It's probably unlikely that users will start submitting reports with no logs and only curl calls, but it's something to think about if that situation does start to happen (I don't think it will, as it takes much more time to craft a curl call than to copy-paste log lines).

**Review by RedXanadu — APPROVED** [review]
Merging now.

**Comment by dune73** [comment]
I see your point on the dev-on-duty needs, @RedXanadu. Let's keep an eye on this.

**Pull request merged**